images/linux/scripts/installers/homebrew.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+################################################################################
+##  File:  homebrew.sh
+##  Desc:  Installs the Homebrew on Linux
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/document.sh
+
+#  Install the Homebrew on Linux
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+echo 'eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)' | sudo tee /etc/profile.d/brew.sh
```

Comment:
This approach will not work for us. You should get environment variables from command `brew shellenv` and put them to the `/etc/environment` file.

```
HOMEBREW_PREFIX="/home/linuxbrew/.linuxbrew"
HOMEBREW_CELLAR="/home/linuxbrew/.linuxbrew/Cellar"
HOMEBREW_REPOSITORY="/home/linuxbrew/.linuxbrew/Homebrew"
```
To append to the PATH env variable, please edit `https://github.com/actions/virtual-environments/blob/master/images/linux/scripts/installers/updatepath.sh` script.

```PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin${PATH+:$PATH}"```

Reply:
Agree, in general.
But I think we should not use `updatepath.sh` file and should update /etc/environments in homebrew.sh. I think everything related to brew should be in the single place

Reply:
https://github.com/actions/virtual-environments/pull/589/commits/37f9820527b9c0f3de0e40d875b3c2a3580f8ac2

Reply:
16.04 build https://github.visualstudio.com/virtual-environments/_build/results?buildId=63719&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=5431112d-2b61-5a5f-7042-ef698f761043
tested on VM https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/417cd125-8a26-41ac-8dcf-30b6d4dbbfed/resourceGroups/github-imagegeneration/providers/Microsoft.Compute/virtualMachines/vsedoli2/overview
for the user `testAdm2` and  another user created from scratch without any additional permissions `u1`